bibliopixel/animation/animation.py

```diff
@@ -5,11 +5,12 @@
 from enum import IntEnum
 
 
-class COMPLETE_REASON(IntEnum):
-    NONE = 0
-    CANCELED = 1
-    MAX_STEPS = 2
-    SELF_COMPLETE = 3
+class State(IntEnum):
+    Ready = 0
```

Comment:
Capitalized means "a class".

You had 'em all upper case before - that works as "a constant".  Or all lower case works just as "a class member".